l2geth/internal/ethapi/api.go

```diff
@@ -1914,8 +1914,9 @@ type EthContext struct {
 	Timestamp   uint64 `json:"timestamp"`
 }
 type RollupContext struct {
-	Index      uint64 `json:"index"`
-	QueueIndex uint64 `json:"queueIndex"`
+	Index         uint64 `json:"index"`
+	QueueIndex    uint64 `json:"queueIndex"`
+	VerifiedIndex uint64 `json:"verifiedIndex"`
```

Comment:
The `VerifiedIndex` represents the last CTC index that was processed via a batch

Reply:
You could add the comment in the code!

Reply:
Added the comment to the code